_validate/validate.py

```diff
@@ -291,12 +316,17 @@ def main():
 		dest="file",
 		help="The json (.json) file containing add-on metadata."
 	)
+	parser.add_argument(
+		dest="APIVersions",
+		help="The JSON file containing valid NVDA API versions."
+	)
 
 	args = parser.parse_args()
 	filename = args.file
+	verFilename = args.versions
```

Comment:
doesn't this need to be changed as well?

```suggestion
	verFilename = args.APIVersions
```